doc/source/user_guide/advanced.rst

```diff
@@ -492,6 +492,7 @@ Using the parameter ``level`` in the :meth:`~DataFrame.reindex` and
 values across a level. For instance:
 
 .. ipython:: python
+   :okwarning:
```

Comment:
would just remove this example

Reply:
*may* instead show a groupby example of this (but then i would put it in the groupby section), you can leave a pointer if you'd like here

Reply:
If ok, I would keep it there, there are level examples above and below this snippet

Reply:
But changed to groupby